system/ee/EllisLab/ExpressionEngine/Controller/Publish/Edit.php

```diff
@@ -588,6 +589,33 @@ private function remove($entry_ids)
 
 		$entry_names = $entries->all()->pluck('title');
 
+		// Remove pages URIs
+		$site_id = ee()->config->item('site_id');
+		$site_pages = ee()->config->item('site_pages');
+
+		if ($site_pages !== FALSE && $entries)
```

Comment:
This can be condensed into a single conditional

```
if ($site_pages !== FALSE && $entries && count($site_pages[$site_id]) > 0)
{
    foreach ($entries->all() as $entry)
	{
    ...
}
```

Reply:
Updated in latest commits.